workspace/apps/pidp/src/app/features/portal/components/portal-card/portal-card.component.scss

```diff
@@ -2,7 +2,7 @@
 
 // Set parent variables
 .viewport-xsmall {
-  --gap: 1rem;
+  --gap: 1.8rem;
```

Comment:
There is still a slight overlay on top of the button
<img width="365" alt="overlay" src="https://github.com/bcgov/moh-pidp/assets/45132887/c91388b2-9cdf-41d5-b4fc-e3b978925b95">